<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVR</class>
 <widget class="QDialog" name="SVR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../feather/activity.svg</normaloff>../feather/activity.svg</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../New folder/ease_ML/feather/help-circle.svg</normaloff>
     <normalon>../feather/help-circle.svg</normalon>../../New folder/ease_ML/feather/help-circle.svg</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>Variables selection</string>
   </property>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dependent variable:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Independent variables:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>161</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>41</width>
      <height>160</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../feather/chevrons-right.svg</normaloff>../feather/chevrons-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../feather/chevron-right.svg</normaloff>../feather/chevron-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../feather/chevron-left.svg</normaloff>../feather/chevron-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../feather/chevrons-left.svg</normaloff>../feather/chevrons-left.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dataset variables:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Hyperparameters</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Random state:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Epsilon:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C (Regulaization):</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Kernel</string>
    </property>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>RBF</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Poly</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sigmoid</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coef0:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Degree: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>radioButton_1</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SVR</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SVR</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
